modules/features/falcon_feature_user/falcon_feature_user.info.yml

```diff
@@ -5,6 +5,7 @@ core: 8.x
 dependencies:
   - field
   - menu_link_config
+  - path
   - user
```

Comment:
Why do we need path module here?

Reply:
@kalabro auto-dependency. But I agree there is no need, fixed.